test/Libraries/DynamoPythonTests/CodeCompletionTests.cs

```diff
@@ -14,6 +15,14 @@ namespace DynamoPythonTests
     [TestFixture]
     internal class SharedCodeCompletionProviderTests : UnitTestBase
     {
+        public override void Setup()
+        {
+            base.Setup();
+            //for some legacy tests we'll need the DSCPython binary loaded manually
```

Comment:
this seems a bit funny... why doesn't https://github.com/DynamoDS/Dynamo/blob/2f9354254b83e9265d45a192a4e6e69c5d629eae/test/DynamoCoreTests/MigrationTestFramework.cs#L12
`GetLibrariesToPreload` work?

Reply:
`GetLibrariesToPreload` is a part of `DynamoModelTestBase` which also starts up dynamo.
CodeCompletionTests do not need to startup dynamo...so I just derived from `UnitTestBase` (which does not have `GetLibrariesToPreload`)